### ISSUE TITLE
single instance not work in the last version?

### ISSUE THREAD

**Issue by vvyoko** [dnGrep/dnGrep#1033]
Redownload and run with default setting

can work
```
F:\Documents\Downloads\dnGrep.4.1.7.0.x86\dnGREP.exe  /f  1
F:\Documents\Downloads\dnGrep.4.1.7.0.x86\dnGREP.exe  /f  2
```

not work (double instance)
```
F:\Documents\Downloads\dnGrep.4.0.115.0.x64\dnGREP.exe /f  1
F:\Documents\Downloads\dnGrep.4.0.115.0.x64\dnGREP.exe /f  2
```

**Comment by doug24** [comment]
Yes, it did stop working in the 4.1.4/4.1.7 releases.  I'll have a fix for the next release.

**Comment by doug24** [comment]
Fixed in [release 4.1.27](https://github.com/dnGrep/dnGrep/releases/tag/v4.1.27.0)
